tools/SetupFlow/DevHome.SetupFlow/Views/QuickstartPlaygroundView.xaml

```diff
@@ -49,6 +50,82 @@
                 <ResourceDictionary Source="ms-appx:///DevHome.SetupFlow/Styles/SetupFlowStyles.xaml" />
                 <ResourceDictionary Source="ms-appx:///DevHome.SetupFlow/Styles/QuickstartStyles.xaml" />
             </ResourceDictionary.MergedDictionaries>
+            <DataTemplate x:Key="ChatResponseMessageTemplate" x:DataType="viewmodels:ChatStyleMessage">
+                <Grid Height="Auto"
+			Margin="4"
+			HorizontalAlignment="Left">
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="Auto" />
+                        <RowDefinition Height="Auto" />
+                    </Grid.RowDefinitions>
+                    <StackPanel Grid.Row="1"
+				Padding="16,8,16,8"
+				CornerRadius="16">
+                        <StackPanel.Background>
+                            <SolidColorBrush Color="WhiteSmoke" />
+                        </StackPanel.Background>
+                        <Border Padding="8,4,8,4"
+									CornerRadius="4">
+                            <Border.Background>
+                                <SolidColorBrush Opacity="0.2"
+							Color="WhiteSmoke" />
+                            </Border.Background>
+                            <StackPanel Orientation="Horizontal"
+										Spacing="8">
+                                <TextBlock MaxWidth="400"

```

Comment:
Lemme get this straight.

A `TextBlock` inside of a `StackPanel` inside of a `Border` inside of a `StackPanel` inside of a `grid`.  Why not a lone `TextBlock`?